google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -745,16 +744,17 @@ public Map<String, String> extract(MutateRowsRequest mutateRowsRequest) {
    *   <li>Add tracing & metrics.
    * </ul>
    */
-  private UnaryCallable<BulkMutation, Void> createBulkMutateRowsCallable() {
-    UnaryCallable<MutateRowsRequest, Void> baseCallable = createMutateRowsBaseCallable();
+  private UnaryCallable<BulkMutation, MutateRowsAttemptResult> createBulkMutateRowsCallable() {
+    UnaryCallable<MutateRowsRequest, MutateRowsAttemptResult> baseCallable =
+        createMutateRowsBaseCallable();
```

Comment:
Can you concatenate createBulkMutateRowsCallable with createMutateRowsBaseCallable? And call it createMutateRowsBaseCallable? nothing seems to use createMutateRowsBaseCallable

Also please add javadoc explaining that the BaseCallable uses an internal result type